httpsupport/http_client.go

```diff
@@ -0,0 +1,102 @@
+package httpsupport
+
+import (
+	"bytes"
+	"io"
+	"io/ioutil"
+	"net/http"
+	"net/url"
+	"strings"
+
+	"context"
+
+	"github.com/goadesign/goa"
+	"github.com/goadesign/goa/client"
+)
+
+// Doer is a wrapper interface for goa client Doer
+type HttpDoer interface {
+	client.Doer
+}
```

Comment:
`HTTPDoer` and not `HttpDoer` .. right ??

Reply:
sure, let me rename that interface

Reply:
fixed in 6133a1a